wordle/README.md

```diff
@@ -0,0 +1,29 @@
+# Wordle in the zkVM
+
+## Quick Start
+
+First, make sure [rustup](https://rustup.rs) is installed. This project uses a [nightly](https://doc.rust-lang.org/book/appendix-07-nightly-rust.html) version of [Rust](https://doc.rust-lang.org/book/ch01-01-installation.html). The [`rust-toolchain`](rust-toolchain) file will be used by `cargo` to automatically install the correct version.
+
+To build all methods and play Wordle within the zkVM, run the following command:
+
+```
+cargo run
+```
+
+Then, start guessing 5 letter words! 
+
+## About the game
+
+The [game](https://github.com/risc0/risc0-rust-examples/blob/main/wordle/host/src/main.rs) consists of two agents: a player and a server. 
+The zkVM enforces that the server plays fairly.
+
+The game begins with the server choosing a secret word from the [wordlist](https://github.com/risc0/risc0-rust-examples/blob/main/wordle/host/src/wordlist.rs), and sending the player the hash of the secret word. 
+
+The player then submits guesses. After each guess, the server returns which letters from the guess match the letters in the secret word. 
+In addition, the server returns a [receipt](https://www.risczero.com/docs/explainers/proof-system/) that attests to the logic of the letter-checking and [includes the hash](https://github.com/risc0/risc0-rust-examples/blob/main/wordle/methods/guest/src/bin/wordle.rs) of the secret word. 
+
+## Ensuring fair play
+
+The player ensures that server isn't cheating using the [`check_receipt` function](https://github.com/risc0/risc0-rust-examples/blob/main/wordle/host/src/main.rs). 
+This function first runs `receipt.verify(WORDLE_ID)` which ensures that the receipt is valid and was generated by the correct binary file. 
+Then, the `check_receipt` function checks that the [journal contents](https://www.risczero.com/docs/explainers/zkvm/) on the receipt include the hash of the secret word. 
```

Comment:
I think it would be useful to be explicit that the hash of the secret word in the journal matches the one sent to the player at the beginning of the game, which ensures that the server isn't changing the secret word during the game. Otherwise, this looks good!